core/src/main/kotlin/di/Module.kt

```diff
@@ -161,7 +192,29 @@ fun ortServerModule(config: ApplicationConfig, db: Database?) = module {
     single { PackageService(get()) }
     single { ProjectService(get()) }
     single { UserService(get()) }
-    single { OrtRunService(get(), get(), get(), get()) }
+    single {
+        OrtRunService(
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get()
```

Comment:
🤣

Reply:
Can the `singleOf(::OrtRunService)` declaration style be applied here?